docs/insights-archive-sample/config/workload_info.json

```diff
@@ -0,0 +1,7166 @@
+            {
+              "imageID": "sha256:5d96dba1fa39b3c52dcebd2b0fb7e5d9033fed92b6abc305099556fdd0934cbc",
+              "firstCommand": "N9KxLV2avCo2"
+            }
+          ]
+        }
+      ]
+    },
+    "tf2L5gEkgV9I": {
+      "count": 3,
+      "terminalCount": 3,
+      "shapes": []
+    },
+    "txOCjzDC13kJ": {
+      "count": 3,
+      "terminalCount": 3,
+      "shapes": []
+    },
+    "u0EhXGvwmLXg": {
+      "count": 5,
+      "terminalCount": 5,
+      "shapes": []
+    },
+    "uIlDfWY07N7C": {
+      "count": 2,
+      "terminalCount": 2,
+      "shapes": []
+    },
+    "uNCbQ734SkXC": {
+      "count": 29,
+      "terminalCount": 29,
+      "shapes": []
+    },
+    "udTSpisujaO4": {
+      "count": 4,
+      "terminalCount": 4,
+      "shapes": []
+    },
+    "uqjLUsp7vAub": {
+      "count": 23,
+      "terminalCount": 23,
+      "shapes": []
+    },
+    "vDGHaQt8x6K_": {
+      "count": 1,
+      "shapes": [
+        {
+          "containers": [
+            {
+              "imageID": "sha256:83fca12e93240b503f88ec192be5ff0d6dfe750f81e8b5ef71af991337d7c584",
+              "firstArg": "zO0oxtw_mcI5"
+            }
+          ]
+        }
+      ]
+    },
+    "vexkieOS4aPU": {
+      "count": 5,
+      "terminalCount": 4,
+      "shapes": [
+        {
+          "containers": [
+            {
+              "imageID": "sha256:14d68ca3d69fceaa6224250c83d81d935c053fb13594c811038c461194599973",
+              "firstCommand": "icTsn2s_EIax"
+            },
+            {
+              "imageID": "sha256:4b4e8823811692d8fd6631c53c64bd9f1c997ac0f48f84d0e1a8e59f7d0bc108",
+              "firstCommand": "N9KxLV2avCo2"
+            },
+            {
+              "imageID": "sha256:653271a4df502eb6ba84256f8e687d2e3aa7ce22911685ab63db4e65c6370d98",
+              "firstCommand": "N9KxLV2avCo2"
+            },
+            {
+              "imageID": "sha256:653271a4df502eb6ba84256f8e687d2e3aa7ce22911685ab63db4e65c6370d98",
+              "firstCommand": "N9KxLV2avCo2"
+            }
+          ]
+        }
+      ]
+    },
+    "vqe5OHD7KGWi": {
+      "count": 67,
+      "shapes": [
+        {
+          "duplicates": 31,
+          "containers": [
+            {
+              "imageID": "sha256:d4ea0e5042b386273d3121a6c35f6e85f93da6c6840fa29fa106276788c8e39f",
+              "firstCommand": "N9KxLV2avCo2"
+            }
+          ]
+        },
+        {
+          "duplicates": 9,
+          "containers": [
+            {
+              "imageID": "sha256:5605ae37ba151283d38773db4150fbee566e19c4d2df42b462add4538ffe8c6f",
+              "firstCommand": "N9KxLV2avCo2"
+            },
+            {
+              "imageID": "sha256:d4ea0e5042b386273d3121a6c35f6e85f93da6c6840fa29fa106276788c8e39f",
+              "firstCommand": "N9KxLV2avCo2"
+            }
+          ]
+        },
+        {
+          "duplicates": 2,
+          "containers": [
+            {
+              "imageID": "sha256:d4ea0e
```

Comment:
Is this from some real cluster? I guess so. The math related to the size of the archive doesn't seem so bad in this case. This file is ~180kB (compact json), which will increase the size of the archive by ~30kB.

Reply:
Yes this is from one of the CI clusters that has a lot of transient workloads.